app/mailers/new_event_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class NewEventMailer < ApplicationMailer
+
+  def notify_volunteer(event)
```

Comment:
unlike other objects that we can just pass in the whole object, mailers can be done as a background job, and should only be fed with the object ID's. Objects tends to be bigger, eating out memory of the system. Also, it could be that the object fed into the background job be stale -- something happened to the record before the background job executed.